pkg/cmd/kubernikusctl/cluster.go

```diff
@@ -0,0 +1,145 @@
+package kubernikusctl
+
+import (
+	"fmt"
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	kubernikus "github.com/sapcc/kubernikus/pkg/client/kubernikus_generated"
+	"github.com/sapcc/kubernikus/pkg/client/kubernikus_generated/operations"
+	"github.com/sapcc/kubernikus/pkg/cmd"
+	"github.com/sapcc/kubernikus/pkg/cmd/printers"
+	"github.com/spf13/cobra"
+	"net/url"
+	//	"github.com/spf13/pflag"
+	// "github.com/gophercloud/gophercloud/openstack/identity/v3/tokens"
+)
+
+type ClusterOptions struct {
+	auth *CredentialsOptions
```

Comment:
We probably need to restructure this. Extract common `AuthOptions` for and reuse in all relevant command